tests/mrp/fake_mrp_atv.py

```diff
@@ -283,6 +283,20 @@ def handle_send_hid_event(self, message, inner):
             self.send(messages.create(0, identifier=message.identifier))
         elif down_press == 0:
             if (use_page, usage) in self.outstanding_keypresses:
+                if (
+                    _convert_key_press(use_page, usage) == 'select'
+                    and self.last_button_pressed == 'home'
+                   ):
+                    msg = messages.device_information_update(
+                        'pyatv', message.identifier, 0)
+                    self.send(msg)
+                if (
```

Comment:
I think this part is no longer valid, right? There should also be some logic handling `WakeDevice`?